### DISCUSSION TITLE
Using Railway.app for production Server

### DISCUSSION THREAD

**Discussion by vergil0gg** [saran-surya/email_auth#84] [Q&A] [answered]
Since Heroku is not accepting the cards I have, I have used **Railway.app** as the alternative at the moment. I have attached the log from [railway.app](http://railway.app/), the output from debug console and the url to the server.

https://emailauthnode-production.up.railway.app/

![0](https://user-images.githubusercontent.com/90584989/213681248-52c17d05-c51a-40a1-84db-1147b95e042d.png)
![unnamed](https://user-images.githubusercontent.com/90584989/213681403-b51f4d80-31f1-4c40-92d0-0582ede2ab5d.png)

**Comment by saran-surya** [comment, marked as answer]
Hi @vergil0gg,
Have you checked regarding the installation of PYTHON on the server ?

Actually will be planning to completely migrate the project to 100% node JS as it will be helpful for single deployments without much hassle.

**Reply by vergil0gg** [reply]
PYTHON not being there was indeed the cause of these errors, added it to the build pack and now it is working. 

**The new issue**: The mail is being sent to the **spam** section. Given that I have not modified the server code at all and it's being sent with the default template. The sender outlook id is fresh and I have not set any mailing rules in the outlook account.

**Reply by saran-surya** [reply]
Okay, but that actually seems a bit wierd. Because when I tried it did get delivered to the inbox with an existing outlook ID. We will need to check regarding this tho.

Thanks for the information.
